openshift/zuul-incoming-webhook-secret-template.yaml

```diff
@@ -0,0 +1,36 @@
+apiVersion: v1
+kind: Template
+metadata:
+  name: zuul-incoming-webhook-secret
+  annotations:
+    description: >
+      This is Zuul Incoming Webhook Secrets - Used by Zuul to talk to OpenShift 
+    openshift.io/display-name: Zuul Incoming Webhook Secrets
+    version: 0.1.0
+    tags: poc,thoth,ai-stacks,zuul
+    template.openshift.io/documentation-url: https://github.com/AICoE/Thoth/
+    template.openshift.io/long-description: >
+      This template defines secrets required to integrate Zuul with Thoth Core Builds.
+    template.openshift.io/provider-display-name: Red Hat, Inc.
+  labels:
+    app: thoth
+    template: zuul-incoming-webhook-secret
+    component: zuul-incoming-webhook
+
+objects:
+- kind: Secret
+  apiVersion: v1
+  type: Opaque
+  data:
+    WebHookSecretKey: "${ZUUL_INCOMING_WEBHOOK_SECRET}"
```

Comment:
I'm not very sure if this could be a problem, but we mostly use "{{ }}" instead of "${ }".
Everything else looks good to me :)  :+1: